apps/explorer/priv/repo/migrations/20241219221922_create_utxoblocks.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Explorer.Repo.Migrations.CreateUTXOBlocks do
```

Comment:
should `priv/repo/xxx` excluded from the source control ? Is it some kind of intermediate product?